### ISSUE TITLE
Breadcrumbs

### ISSUE THREAD

**Issue by porchlight** [LonDUG/menu_position#24]
Make sure the breadcrumb of the Menu Position Rule Parent shows up.

**Comment by porchlight** [comment]
Need to discuss this more, or not do this for the initial release since D8 doesn't handle breadcrumbs at all. The Menu Breadcrumb module currently being ported to D8 would be useful to leverage possible, but would require a small patch to allow our module to work with it. I don't know.

**Comment by midnightLuke** [comment]
We talked about this at length last night but essentially I got cold feet because Drupal doesn't really handle breadcrumbs this way anymore.  There are issues in the on Drupal.org related to this issue in core (https://www.drupal.org/node/2098323, https://www.drupal.org/node/2070651, https://www.drupal.org/node/2333845), but the general consensus seems to be that "menu-based breadcrumbs are hard" and "we can't please everyone so contrib to the rescue" 😛 .

**Comment by ckaotik** [comment]
I agree that this is out of scope of the module. As we're already providing menu links that inherit from `MenuLinkBase`, this should work out of the box for any proper menu-based implementation.
